cylc/flow/scheduler.py

```diff
@@ -546,9 +543,10 @@ async def start_servers(self):
         self.pub_port = self.publisher.port
 
     async def log_start(self):
+        # TODO: remove or reimplement n_restart when DB has a table for the
```

Comment:
Could/should this have a ticket as well as/instead of a TODO comment? If you wish to leave the comment perhaps it should reference the ticket?

Reply:
Oops, that should've been removed